src/components/card-decks.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import { Col } from 'react-bootstrap';
+import DottedBox from '../components/icons/dotted-box';
+
+const FullCard = ({ card }) => (
+  <div className="card rounded shadow-sm p-2 mt-4">
+    <Link to={card.fields.path}>
+      <DottedBox className="opacity-1 mt-3 ml-3" width="100" height="100" />
+    </Link>
+    <div className="card-body">
+      <h3 className="card-title balance-text">
+        <Link to={card.fields.path}>{card.frontmatter.title}</Link>
+      </h3>
+
+      <p className="card-text">{card.frontmatter.description}</p>
+
+      {card.children.map(child => (
+        <Link
+          key={child.fields.path}
+          to={child.fields.path}
+          className="btn btn-link btn-block text-left p-0"
+        >
+          {child.frontmatter.title}
+        </Link>
+      ))}
+    </div>
+  </div>
+);
+
+const SimpleCard = ({ card }) => (
+  <div className="card rounded shadow-sm p-2 mt-4">
+    <div className="card-body">
+      <h3 className="card-title balance-text">
+        <Link className="stretched-link" to={card.fields.path}>
+          {card.frontmatter.title}
+        </Link>
+      </h3>
+
+      <p className="card-text">{card.frontmatter.description}</p>
+    </div>
+  </div>
+);
+
+const CardDecks = ({ cards, colSize = 4, cardType = 'simple' }) => {
+  return (
+    <div className="card-deck row no-gutters">
+      {cards.map(card => {
+        switch (cardType) {
+          case 'full': {
+            return (
+              <Col key={card.fields.path} md={colSize} className="d-flex">
```

Comment:
I tried to move the `Col` up above the switch statement, but when I do that I get this error...

![Screen Shot 2020-05-29 at 12 26 41 PM](https://user-images.githubusercontent.com/2780987/83282926-56240400-a1a8-11ea-8505-aefbd1f1d1bf.png)

Any idea? I'm a little confused why it doesn't work.